packages/office-ui-fabric-react/src/components/Dialog/Dialog.tsx

```diff
@@ -37,6 +32,13 @@ export class Dialog extends BaseComponent<IDialogProps, {}> {
   constructor(props: IDialogProps) {
     super(props);
 
+    Modal.defaultProps = {
```

Comment:
This seems very, very bad.

Reply:
We should not be modifying the static default props of Modal from Dialog!

Reply:
Yeah help us! The other way wasn't working.

Reply:
I couldn't get any of the modalProps inside defaultProps to take effect when I was helping Ben test

Reply:
Having a dark overlay for dialog, which is different from our products, is very, very bad. #FailingFabricExample. Sad!

Reply:
Help us then @dzearing!

Reply:
...uh, so then why don't you flip the one boolean to false? Instead of redoing how default modalProps are defined?

```
 modalProps {
      isDarkOverlay: true,
```